easybuild/easyblocks/l/llvm.py

```diff
@@ -376,6 +377,23 @@ def __init__(self, *args, **kwargs):
         self._cmakeopts = {}
         self._cfgopts = list(filter(None, self.cfg.get('configopts', '').split()))
 
+    @property
+    def gcc_prefix(self):
+        """Return the GCC prefix (versioned folder in <gcc_root>/lib)."""
+        self._set_gcc_prefix_probs()
+        return self._gcc_prefix
+
+    @property
+    def gcc_root(self):
+        """Return the GCC root folder from dependencies/toolchain."""
+        self._set_gcc_prefix_probs()
+        return self._gcc_root
+
+    def _set_gcc_prefix_probs(self):
+        """Set properties of currently loaded GCC installation"""
+        self._gcc_root, self._gcc_prefix = self._get_gcc_prefix()
```

Comment:
I would put this inside an if as it was done originally.
I do not foresee any situation where these variables would change during the build process

Reply:
This is a method only used on access of the properties to set the "backing storage". So the if is outside this function

I agree they won't change, so there is no harm if this method is (wrongly) called again

Reply:
There is no `if` either inside `gcc_prefix` nor `gcc_root` so the method `self._get_gcc_prefix()` will be called every time one of these 2 properties are requested